requirements_test.txt

```diff
@@ -5,3 +5,4 @@ pytest-rerunfailures
 pytest-cov
 numpy
 autograd
+requests
```

Comment:
nit: HTTP clients constantly break, I'd suggest adding a constraint on the versions 😄

Reply:
OK